pom.xml

```diff
@@ -3,9 +3,10 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.mesos</groupId>
-  <artifactId>storm</artifactId>
-  <version>0.9.6</version>
-  <name>storm</name>
+  <artifactId>storm-parent</artifactId>
+  <version>0.1.0</version>
```

Comment:
I'm thinking we'll want to use a `-SNAPSHOT` suffix in the version, allowing us to directly use the `maven release` plugin.  Or we could fake its use by manually rev'ing the version.  Thoughts?